src/content/changes/highlights/2024-07-21.md

```diff
@@ -0,0 +1,186 @@
+- [cluster-azure](https://github.com/giantswarm/cluster-azure)
+    - [0.16.1](https://github.com/giantswarm/cluster-
+azure/compare/v0.16.0...v0.16.1)
+        - Respect `global.apps.externalDnsPrivate` to overwrite configuration of `external-dns-private` app.
+- [cloud-provider-cloud-director-app](https://github.com/giantswarm/cloud-provider-cloud-director-app)
+    - [0.3.0](https://github.com/giantswarm/cloud-provider-cloud-director-
+app/compare/v0.2.11...v0.3.0)
+        - Rollback CPI from `1.6.0` to `1.5.0` due to IP Spaces incompatibility.
+- [cluster-cloud-director](https://github.com/giantswarm/cluster-cloud-director)
+    - [0.56.1](https://github.com/giantswarm/cluster-cloud-
+director/compare/v0.56.0...v0.56.1)
+        - Rollback CPI from `1.6.0` to `1.5.0` due to VCD version incompatibility.
+    - [0.56.0](https://github.com/giantswarm/cluster-cloud-
+director/compare/v0.55.0...v0.56.0)
+        - Bump Kubernetes to `1.27.14`.
+    - [0.55.0](https://github.com/giantswarm/cluster-cloud-
+director/compare/v0.54.1...v0.55.0)
+        - Bump Kubernetes to `1.26.14`.
+- [cluster-vsphere](https://github.com/giantswarm/cluster-vsphere)
+    - [0.56.1](https://github.com/giantswarm/cluster-
+vsphere/compare/v0.56.0...v0.56.1)
+        - Unpause Cluster resource as part of cleanup hook after deletion in order to prevent leftover resources.
+- [default-apps-cloud-director](https://github.com/giantswarm/default-apps-cloud-director)
+    - [0.9.0](https://github.com/giantswarm/default-apps-cloud-
+director/compare/v0.8.0...v0.9.0)
+        - Update `cert-exporter` to v2.9.1.
+        - Update `cert-manager-app` to v3.8.0.
+        - Update `k8s-dns-node-cache-app` to v2.8.1.
+        - Update `net-exporter to v1.21.0`
+        - Update `observability-bundle` to v1.4.0.
+        - Update `security-bundle` to v1.7.1.
+        - Update `teleport-kube-agent-app` to v0.9.2.
+        - Update `vertical-pod-autoscaler-app` to v5.2.4.
+- [azure-private-endpoint-operator](https://github.com/giantswarm/azure-private-endpoint-operator)
+    - [0.2.3](https://github.com/giantswarm/azure-private-endpoint-
+operator/compare/v0.2.2...v0.2.3)
+
+- [cluster](https://github.com/giantswarm/cluster)
+    - [0.36.0](https://github.com/giantswarm/cluster/compare/v0.35.0...v0.36.0)
+      This release removes the `CronJobTimeZone` feature gate as it becomes stable
+      and is included in Kubernetes v1.29.
+      For Kubernetes <v1.29, you will need to re-enable it using the respective
+      values.
+        - Feature Gates: Remove `CronJobTimeZone`. ([#267](https://github.com/giantswarm/cluster/pull/267))
+- [cluster-autoscaler-app](https://github.com/giantswarm/cluster-autoscaler-app)
+    - [1.29.3-gs1](https://github.com/giantswarm/cluster-autoscaler-
+app/compare/v1.28.5-gs1...v1.29.3-gs1)
+        - Chart: Update to upstream v1.29.3. ([#279](https://github.com/giantswarm/cluster-autoscaler-app/pull/279))
```

Comment:
@yulianedyalkova These are from Turtles. Please adapt the content to the standard format (name, version, description)
[example](https://raw.githubusercontent.com/giantswarm/docs/main/src/content/changes/highlights/2024-02-15.md)

Reply:
@pipo02mix I'm confused, I see this file is autogenerated - can't we just adapt the automation to produce it in the right format rather than ping all POs to manually adapt?

Reply:
Yeah, I can work with the format. But we don't write changelog entries considering the customer, so the message can be too technical and not focus on what it brings to the end user. IMO pinging POs every two weeks to adapt released apps is not a lot. Now there are more entries than usual because we did not share highlights for a month.

Reply:
Ah gotcha, so it's more about rephrasing it to be customer-friendly rather than the format itself. Will take care of it then :)

Reply:
yeah, thx